Rust/README.md

```diff
@@ -0,0 +1,20 @@
+# Rust
+[Rust](https://www.rust-lang.org) is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety.
+
+## Installation
+For installation it's advised to use `rustup` as it will (among other things)
+allow you to switch between versions of Rust without having to download
+anything additional.
+
+```sh
+$ brew install rustup
```

Comment:
I used `$ brew ...` here but I see that we are sometimes using just `brew ...`.

What do you think we should use? Personally I'm fine with both.

Reply:
I think both are okay and completely okay with the tradeoff. We could just leave it to the author for now.